code/go/0chain.net/smartcontract/dbs/event/user.go

```diff
@@ -0,0 +1,121 @@
+package event
+
+import (
+	"0chain.net/chaincore/state"
+	"errors"
+	"fmt"
+	"gorm.io/gorm"
+)
+
+type User struct {
+	gorm.Model
+	TransactionID string        `json:"transaction_id"`
+	Balance       state.Balance `json:"balance"`
+	UserID        string        `json:"user_id"`
+
+	// Non sql fields
+	Amount state.Balance `gorm:"-"`
```

Comment:
Is it being stored as array? Not familiar with the `-` notation in the gorm what does it mean, will it not create a column for this?

Reply:
Yes